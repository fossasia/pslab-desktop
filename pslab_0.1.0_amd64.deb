version https://git-lfs.github.com/spec/v1
oid sha256:b0c9f437108df24f1ff5524763d78db5eeb1b3ef9266046996dc8d6814edb897
size 102670990
