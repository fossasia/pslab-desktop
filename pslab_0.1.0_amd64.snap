version https://git-lfs.github.com/spec/v1
oid sha256:17c8e320ae189de0d144a4b4b8b7a1089ae8e8dffebf59a3a3aded1aedcc55ea
size 91783168
